core/cmd/lambda/main.go

```diff
@@ -67,14 +76,14 @@ func (h corsHeaders) setHeaders(resp events.APIGatewayProxyResponse) events.APIG
 
 func handler(
 	clientModel core.ClientInputToGraph, clientDiagram core.ClientGraphToDiagram, corsHeaders corsHeaders,
+	clientStorage core.ClientStorage,
 ) func(
 	ctx context.Context, req events.APIGatewayProxyRequest,
 ) (events.APIGatewayProxyResponse, error) {
 	return func(
 		ctx context.Context, req events.APIGatewayProxyRequest,
 	) (events.APIGatewayProxyResponse, error) {
-		// FIXME: add proper sink to preserve user's requests for model fine-tuning
-		log.Println(req.Body)
+		defer func() { _ = clientStorage.Close(ctx) }()
```

Comment:
**question** why don't we log the error instead of silent it?

Reply:
@diegohordi good proposal. What for though? Errors shall be treated, what treatment would you suggest for a connection failed to be closed?